presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -3972,7 +3972,7 @@ public void testCollectColumnStatisticsOnCreateTable()
                         "('p_varchar', 0E0, 0E0, 0E0, null, null, null), " +
                         "(null, null, null, null, 0E0, null, null)");
 
-        assertUpdate(format("DROP TABLE %s", tableName));
+        assertUpdate("DROP TABLE " + tableName);
```

Comment:
Unrelated change, please extract to separate commit (within this PR), eg `Improve formatting`

Reply:
Done